src/main/java/core/basesyntax/service/RegistrationServiceImpl.java

```diff
@@ -5,10 +5,99 @@
 import core.basesyntax.model.User;
 
 public class RegistrationServiceImpl implements RegistrationService {
+    private static final int ZERO = 0;
+    private static final int MIN_AGE = 18;
+    private static final int MAX_AGE = 120;
+    private static final int MIN_PASSWORD_LENGTH = 6;
     private final StorageDao storageDao = new StorageDaoImpl();
 
     @Override
     public User register(User user) {
-        return null;
+        checkUser(user);
+        checkAgeNull(user);
+        checkAgeLess18(user);
+        checkAgeBigger100(user);
+        checkAgeZero(user);
```

Comment:
you don't need to check age less than 0, it doesn't make sense, cause user age cannot be less than 18